README.md

```diff
@@ -2,100 +2,128 @@
 
 [![Build Status](https://secure.travis-ci.org/jzaefferer/commitplease.png)](http://travis-ci.org/jzaefferer/commitplease)
 
-This [node.js](http://nodejs.org/) module validates git commit messages while you commit, according to the rules specified in the [jQuery Commit Guidelines](http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines). These are pretty generic, so this module should be useful for various projects that care about commit message formats.
+This [node.js](http://nodejs.org/) module makes sure your commit
+messages consistently follow one of these style guides:
```

Comment:
Don't need to limit lines in prose, this linebreak can be dropped, see also http://contribute.jquery.org/style-guide/prose/